addons/im_livechat/models/ir_autovacuum.py

```diff
@@ -1,12 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import api, models
-
-class AutoVacuum(models.AbstractModel):
-    _inherit = 'ir.autovacuum'
-
-    @api.model
-    def power_on(self, *args, **kwargs):
-        self.env['mail.channel'].remove_empty_livechat_sessions()
```

Comment:
`remove_empty_livechat_sessions` should probably get tagged too?

Reply:
indeed